### PR TITLE
Sidm 3296 waf password configure test

### PR DESCRIPTION
**Pull request by sudhasane**
https://tools.hmcts.net/jira/browse/SIDM-3296
**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ X] No
```

### CODE REVIEW

File: src/test/js/self_registration_test.js [hunk 1]
```diff
@@ -112,15 +114,18 @@ Scenario('@functional @selfregister I can self register', async (I) => {
     I.fillField('#password1', TestData.PASSWORD);
     I.fillField('#password2', TestData.PASSWORD);
     I.click('Continue');
-    I.waitForText('Account created', 20, 'h1');
+    I.wait(30);
```
Comment (shravanmechineni) [inline src/test/js/self_registration_test.js]: I don't think 30 secs wait required here as this scenario passed in the current preview build without this.

File: src/test/js/self_registration_test.js [hunk 2]
```diff
@@ -112,15 +114,18 @@ Scenario('@functional @selfregister I can self register', async (I) => {
...
+    I.waitForText('Account created', 40, 'h1');
     I.see('You can now sign in to your account.');
     I.amOnPage(loginPage);
     I.seeInCurrentUrl("state=selfreg");
+    I.wait(30);
```
Comment (shravanmechineni) [inline src/test/js/self_registration_test.js]: I don't think 30 secs wait required here as this scenario passed in the current preview build without this.

File: src/test/js/self_registration_test.js [hunk 3]
```diff
@@ -112,15 +114,18 @@ Scenario('@functional @selfregister I can self register', async (I) => {
...
     I.waitForText('Sign in or create an account', 20, 'h1');
     I.fillField('#username', email);
     I.fillField('#password', TestData.PASSWORD);
     I.interceptRequestsAfterSignin();
     I.click('Sign in');
+    I.wait(30);
```
Comment (shravanmechineni) [inline src/test/js/self_registration_test.js]: I don't think 30 secs wait required here as this scenario passed in the current preview build without this.

File: src/test/js/login_user_functional_test.js [hunk 4]
```diff
@@ -67,4 +67,4 @@ Scenario('@functional @login As a citizen user I can login with email in upperca
 
     I.resetRequestInterception();
 
-}).retry(TestData.SCENARIO_RETRY_LIMIT);
\ No newline at end of file
+}).retry(TestData.SCENARIO_RETRY_LIMIT);
```
Comment (tbamido) [inline src/test/js/login_user_functional_test.js]: not sure what changed here 🤷‍♂

**Pull request merged**